utils/sql-db-seeder/src/mongo-db-client/mongo-db-client.ts

```diff
@@ -0,0 +1,21 @@
+import z from 'zod';
+import { MongoDbClient } from '@ukef/dtfs2-common/mongo-db-client';
+
+const mongoDbConfigSchema = z.object({
+  MONGODB_URI_QA: z.string(),
+  MONGO_INITDB_DATABASE: z.string(),
+});
+
+/**
+ * Gets the mongo db client
+ * @returns The mongo db client
+ */
+const getMongoDbClient = (): MongoDbClient => {
+  const mongoDbConfig = mongoDbConfigSchema.parse(process.env);
```

Comment:
This is the approach we use elsewhere for parsing env variables, and there is no real performance overhead in using it due to the relatively small size of `process.env`, so I'm inclined to keep it for now and ask for a DTFS ticket to be added if this wants to be changes across services and utils

Reply:
I recommend having such changes in a separate PR, this has nothing to do with adding "reconciliation data to payment details".

Reply:
@ttbarnes Which changes are you referring to? The updates for the seeder are required with the payment details changes so as to not break the seeder for local development

Reply:
I can see there are three instances and should not take long to update, could you please update all the four instances (including this one) to following please:

```typescript
// Fetch env variables
const { MONGODB_URI_QA, MONGO_INITDB_DATABASE } = process.env;

// Required environment variable
const mongoEnvs = {
  MONGODB_URI_QA,
  MONGO_INITDB_DATABASE,
};

const mongoDbConfig = mongoDbConfigSchema.parse(mongoEnvs);
```

Above approach encourages `Composition over Inheritance` which promotes flexibility and decoupling.

Reply:
I don't think the above change is correctly applying the principle of composition of inheritance as this doesn't have any links to OOP, but I will make the change with the caveat that you are still loading the entire `process.env` object at the destructing line, and that this code is actually harder to maintain as, if you update the `MONGODB_URI_QA` name, for instance, you would have to update both the assignment and the schema

Reply:
Resolving this as this client is consistent with other mongo DB clients in the `utils` directory. Changing this client will cause misalignment.

I think i've mentioned before - we could/should use the same, single mongo DB client instead of duplicating it.

We should create a single DRY instance and tweak the env vars approach mentioned by @abhi-markan in a future ticket/PR.